runtime/serde/serde-json/common/src/aws/smithy/kotlin/runtime/serde/json/JsonStreamReader.kt

```diff
@@ -34,14 +34,14 @@ interface JsonStreamReader {
 fun jsonStreamReader(payload: ByteArray): JsonStreamReader = JsonLexer(payload)
 
 // return the next token and require that it be of type [TExpected] or else throw an exception
-internal inline fun <reified TExpected : JsonToken> JsonStreamReader.nextTokenOf(): TExpected {
+inline fun <reified TExpected : JsonToken> JsonStreamReader.nextTokenOf(): TExpected {
     val token = this.nextToken()
     requireToken<TExpected>(token)
     return token as TExpected
 }
 
 // require that the given token be of type [TExpected] or else throw an exception
-internal inline fun <reified TExpected> requireToken(token: JsonToken) {
+inline fun <reified TExpected> requireToken(token: JsonToken) {
     if (token::class != TExpected::class) {
         throw DeserializationException("expected ${TExpected::class}; found ${token::class}")
     }
```

Comment:
**Question**: We thought these should be `internal` previously. Do we intend for them to be freely used now or should they be decorated with `@InternalApi`?

Reply:
I'll add `@InternalApi`